granulate_utils/containers/docker.py

```diff
@@ -34,16 +35,21 @@ def get_container(self, container_id: str, all_info: bool) -> Container:
     def get_runtimes(self) -> List[str]:
         return ["docker"]
 
-    @staticmethod
-    def _create_container(container: docker.models.containers.Container) -> Container:
+    @classmethod
+    def _create_container(cls, container: docker.models.containers.Container) -> Container:
         pid: Optional[int] = container.attrs["State"].get("Pid")
+        created = isoparse(container.attrs["Created"])
+        assert created is not None
+        started_at = isoparse(container.attrs["State"]["StartedAt"])
```

Comment:
If not started yet?

Reply:
Oh fuck I had logic to handle it here [73a00e4](https://github.com/Granulate/granulate-utils/pull/129/commits/73a00e4e560936979a7f53752fd693c3966125f8)